### PR TITLE
Compiled figure updates

### LINKED ISSUE #66: Add individual panel labels for Figure 2 QC report examples
I was reviewing https://github.com/AlexsLemonade/ScPCA-manuscript/pull/36, and I agree with @sjspielman's comment:

> While writing this, I wondered whether we actually want to make these real panels in their own right, as panels B-G, which may make it easier to read.

### LINKED ISSUE #67: Add individual panel labels for Figure S2B ADT plot report examples
As originally mentioned in #66, we should add individual panel labels to S2B.

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #66 
Closes #67 

This PR updates the compiled versions of Fig 2 and S2:

#### Figure 2
I broke down panel B into panels B-G and updated the compiled figure. To improve the alignment in the compiled figure, I also separated out the final panel G, which was a little too tall in the previous compilation to easily place the panel label. This is now exported as its own PDF panel.
As such:
- `figures/pdfs/Fig2B_mini-qc-plots.pdf` is now gone
- Instead we now have `figures/pdfs/Fig2B-F_mini-qc-plots.pdf` and `figures/pdfs/Fig2G_mini-qc-hvgs.pdf`
  - Associated script updates in `scripts/Fig2B-G_qc-plots.R`

#### Figure S2
I broke down panel B into panels B-D and updated the compiled figure.

- `figures/pngs/FigS2B_adt-plots.png` is now named `figures/pngs/FigS2B-D_adt-plots.png`
  - associated script change in `scripts/FigS2B-D_adt-plots.R`

The `scripts/README.md` is also updated accordingly.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Before running this script, you must run `figure_setup/sync-metadata.R`.
 3. `TableS1_modality-summary.R`: This script is used to generate supplemental Table 1, which contains a summary of the types of libraries found in each project.
 Before running this script, you must run `figure_setup/sync-metadata.R`.
 
-4. `Fig2B_qc-plots.R`: This script is used generate Figure 2B, which includes simplified and miniature versions of the plots found in the main QC report included with each sample download.
+4. `Fig2B-G_qc-plots.R`: This script is used generate panels Figure 2B-G, which includes simplified and miniature versions of the plots found in the main QC report included with each sample download.
```
Comment (allyhawkins) [inline scripts/README.md:51]:
```suggestion
4. `Fig2B-G_qc-plots.R`: This script is used to generate panels Figure 2B-G, which includes simplified and miniature versions of the plots found in the main QC report included with each sample download.
```

**Review by allyhawkins — COMMENTED** [review]
When I opened the pdf in photoshop locally the font is still Helvetica for the mini panels. Can you change it back to Arial please?

**Comment by sjspielman** [comment]
Arial-ified, great catch!!

**Review by allyhawkins — APPROVED** [review]
🚀

**Pull request merged**